cmake/modules/HandleLLVMOptions.cmake

```diff
@@ -56,9 +56,10 @@ int main() { return (float)x; }"
 endif()
 
 if( LLVM_ENABLE_ASSERTIONS )
-  set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -DDBG") # HLSL Change
-  set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -UNDEBUG") # HLSL Change
-  if (0) # HLSL Change Starts
+  # MSVC doesn't like _DEBUG on release builds. See PR 4379.
```

Comment:
What PR is this referencing?  This PR number doesn't exist yet here.

Reply:
This is referencing the LLVM bug report (why bugzilla used to call them PRs I have no idea). The URL is: https://bugs.llvm.org/show_bug.cgi?id=4379